components/cluster.py

```diff
@@ -245,14 +341,42 @@ def get_plot(graph, positions):
     ))
 
     fig.add_trace(go.Scatter(
-        x=Xe,
-        y=Ye,
+        x=X_foreshocks,
+        y=Y_foreshocks,
+        hovertext=Hover_foreshocks,
+        hoverinfo="text",
+        mode='markers',
+        name='Foreshocks',
+        marker=dict(
+            symbol='circle-dot',
+            size=18,
+            color='#ffd400',
+            line=dict(color='rgb(50,50,50)', width=1)
+        )
+    ))
+    fig.add_trace(go.Scatter(
+        x=X_aftershocks,
+        y=Y_aftershocks,
+        mode='markers',
+        name='Aftershocks',
+        hovertext=Hover_aftershocks,
+        hoverinfo="text",
+        marker=dict(
+            symbol='circle-dot',
+            size=18,
+            color='#0055ff',
+            line=dict(color='rgb(50,50,50)', width=1)
+        )
+    ))
+    fig.add_trace(go.Scatter(
+        x=[max_time],
+        y=[max_magnitude],
         mode='markers',
-        name='Earthquake',
+        name='Mainshock',
         marker=dict(
             symbol='circle-dot',
             size=18,
-            color='#6175c1',
+            color='#ba0000',
             line=dict(color='rgb(50,50,50)', width=1)
```

Comment:
There is duplicated code that could be transferred under one function for better readability

Reply:
Included get_figure function in the last commit.